content/concepts/transports/webrtc.md

```diff
@@ -101,17 +101,85 @@ Authenticity is achieved by succeeding the
 
 <!-- TO ADD DIAGRAM -->
 
-### Coming soon: Browser-to-Browser
+### WebRTC W3C (Hole punching to achieve Browser-to-Browser connectivity)
 
-Eventually, libp2p will have support for communication between two
-browsers.
+Thanks to js-libp2p and rust-libp2p (complied to Wasm), libp2p can run in the browser environment.
+However, browsers impose certain restrictions on application code (such as libp2p browser nodes).
+Applications are sandboxed and face constraints on security and networking.
+For instance, browsers do not permit direct access to raw network sockets.
+Additionally, it's a sure bet that libp2p browser nodes will be behind a NAT/firewall.
+Due to these restrictions, browser nodes cannot listen for incoming connections,
+and as a result, they cannot communicate with other browser nodes.
 
-The technical specification and initial implementations of WebRTC
-Browser-to-Browser connectivity is planned for release in early 2023.
-Track the progress [here](https://github.com/libp2p/specs/issues/475).
+Thankfully, libp2p solves this problem and enables browser node to browser node connectivity by supporting a transport called WebRTC W3C.
+
+{{< alert icon="" context="info">}}
+Due to the restrictions mentioned above, libp2p browser nodes are private nodes (meaning they cannot be dialed from outside of their local network).
```

Comment:
Not really. They're private nodes because people run their browsers in their home / corporate networks, and those networks almost always use NATs. In principle, you could run your browser on a machine that has a public address.

Reply:
I'll make this clearer